app/views/catarse_bootstrap/errors/not_found.html.slim

```diff
@@ -1,8 +1,9 @@
 = render 'devise/shared/header'
-.row
-  .large-5.large-centered.columns
-    .box.animated.fadeIn
-      h2.text-center= t('.title')
-      hr
-      .large-11.columns.large-centered
-        p= t('.text')
+.w-section
+	.w-container.u-text-center
```

Comment:
Some indentation problem here @thiagocatarse you should adjust your text editor settings to use soft tabs with 2 spaces.